diff -uNr gcc-11.2.1-20210728/libiberty/fibheap.c gcc-11.2.1/libiberty/fibheap.c
--- gcc-11.2.1-20210728/libiberty/fibheap.c	2021-07-28 17:09:49.000000000 +0800
+++ gcc-11.2.1/libiberty/fibheap.c	2021-08-20 11:45:47.881997656 +0800
@@ -22,9 +22,7 @@
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
-#ifdef HAVE_LIMITS_H
 #include <limits.h>
-#endif
 #ifdef HAVE_STDLIB_H
 #include <stdlib.h>
 #endif
