From 2ea5d5136fdeaa49ffba7ec8ccb20fb92df4ff20 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20K=2E=20H=C3=BCttel?= <dilfridge@gentoo.org>
Date: Wed, 17 Jan 2018 01:02:19 +0100
Subject: [PATCH 08/94] Gentoo: Disable test that fails because of the
 gethostbyname4_r revert

---
 nss/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/nss/Makefile b/nss/Makefile
index 66fac7f5b8..fb77cc688e 100644
--- a/nss/Makefile
+++ b/nss/Makefile
@@ -63,7 +63,6 @@ xtests			= bug-erange
 # Tests which need libdl
 ifeq (yes,$(build-shared))
 tests += tst-nss-files-hosts-erange
-tests += tst-nss-files-hosts-multi
 tests += tst-nss-files-hosts-getent
 endif
 
-- 
2.21.0

